site/content/docs/5.3/utilities/link.md

```diff
@@ -87,14 +87,20 @@ Just like the `.link-opacity-*-hover` utilities, `.link-offset` and `.link-under
 
 [Colored link helpers]({{< docsref "/helpers/colored-links/" >}}) have been updated to pair with our link utilities. Use the new utilities to modify the link opacity, underline opacity, and underline offset.
 
-<!--Boosted mod: feature limited to primary and light, we don't loop over "theme-colors"-->
+<!--Boosted mod: add bg-dark on the links which need it to ensure contrast-->
 {{< example >}}
-<p><a href="#" class="link-primary link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Primary link</a></p>
-<p><a href="#" class="link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Light link</a></p>
+{{< colored-links.inline >}}
+{{- range (index $.Site.Data "theme-colors") }}
+<p><a href="#" class="link-{{ .name }}{{ with .contrast_color }} bg-{{ . }}{{ end }} link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">{{ .name | title }} link</a></p>
```

Comment:
I'd say you could create a PR for Bootstrap to make this same change for consistency between the "utilities/colors" and "utilities/link" rendering. Hope it works well for dark mode too.

Reply:
ok, I'll do that :-)